config/routes.rb

```diff
@@ -68,6 +68,9 @@
   resources :roles
 
   resources :functions, only: %i(create destroy index update)
+  resources :members
+  resources :addresses
```

Comment:
we could nest `:addresses` under `:members`, `:events`, `:teams`.

URL-wise the prefered solution is:
* `/players/[:id]/address` (note the singular)
* `/events/[:id]/address` (note the singular)
* `/teams/[:id]/addresses`

I guess this could be achieved by:
```ruby
resources :players do
  resource :address, shallow: true # note the singular
end
resources :events do
  resource :address, shallow: true # note the singular
end
resources :teams do
  resources :addresses, shallow: true
end
```

what do you think?